core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -61,9 +61,18 @@ export interface ContributionsSummary {
   actionButtonTitle: string;
 }
 
+export interface OpportunitiesDicts {
```

Comment:
```suggestion
export interface Opportunity {
```

Assuming there are no naming conflicts.

Reply:
done